index.js

```diff
@@ -241,34 +250,35 @@ function adapter(uri, opts, onNamespaceInitializedCallback)
         Adapter.prototype.broadcast.apply(this, args);
     };
 
-
     /**
-     * Subscribe client to room messages.
+     * Adds a socket to a list of room.
      *
-     * @param {String} id Client ID
-     * @param {String} room
-     * @param {Function} fn (optional)
+     * @param {String} socket id
+     * @param {String} rooms
+     * @param {Function} callback
      * @api public
      */
 
-    AMQPAdapter.prototype.add = function (id, room, fn)
+    AMQPAdapter.prototype.addAll = function (id, rooms, fn)
```

Comment:
should there still be a single `AMQPAdapter.prototype.add` function?

Reply:
The single one now just calls the addAll in the new version and seems like it's there for backwards compatibility only because the socket.js itself only calls addAll judging by the source code.